apps/web/src/pages/JobPosterTemplates/JobPosterTemplatesPage/JobPosterTemplatesPage.tsx

```diff
@@ -308,8 +308,8 @@ const JobPosterTemplatesPage = () => {
         <p data-h2-margin="base(x1 0 x2 0)">
           {intl.formatMessage({
             defaultMessage:
-              "This library contains a series of templates for common jobs across the communities we support. These templates include suggested information and recommendations for elements such as job title, common tasks in the role, essential and asset skills, and more. Feel free to browse the full list, or, if you have an idea of the type of role you're hiring for, you can use the search bar or filters to find templates that align with the job’s specific details.",
-            id: "8dUPjb",
+              "This library contains a series of templates for common jobs across the communities that GC Digital Talent supports. The templates include suggested information and recommendations for elements such as the job title, common tasks in the role, and essential and asset optional skills. Browse the full list or use the search bar and filters to find templates that align with the job’s specific details.",
```

Comment:
So, this is what was specified in the ticket but I'm finding the following a bit awkward.

> recommendations for...and essential and asset optional skills.

1.  Two "and"s within short succession of each other, which isn't that bad just makes for odd reading.
2. Reading "asset optional" seems odd. Could just be me but I feel like swapping the words could help "optional asset". Or, removing one of them since we usually use asset and optional interchangeably. 

@NienkeBr Thoughts?

Reply:
Okay, https://github.com/GCTC-NTGC/gc-digital-talent/pull/11609/commits/87a358990c82819a44301d32fad2f66816b3e2d4